### ISSUE TITLE
Examples are not working due to missing throttling configurations

### ISSUE THREAD

**Issue by mtyurt** [terraform-aws-modules/terraform-aws-apigateway-v2#13]
I'm testing this with terraform 0.14.3, aws provider 3.19.0, and the module version 0.5.0. 

After applying the examples, I have received the response:
```
HTTP/1.1 429 Too Many Requests
{
    "message": "Too Many Requests"
}
```

Then I changed the throttling configuration manually through the WEB console, to see if it was going to work. The requests then returned `200` successfully. To see what was changed, I ran `terraform plan`, and here is the difference:
```
  ~ resource "aws_apigatewayv2_stage" "default" {
        id              = "$default"
        name            = "$default"
        # (7 unchanged attributes hidden)

      ~ default_route_settings {
          - throttling_burst_limit   = 100 -> null
          - throttling_rate_limit    = 100 -> null
            # (2 unchanged attributes hidden)
        }
        # (1 unchanged block hidden)
    }
```

I saw that the throttling configurations are commented out. Is this something expected? Am I missing a configuration here? Here is my configuration:

```hcl
module "api_gateway" {
  source = "terraform-aws-modules/apigateway-v2/aws"

  name          = "http-lambda-example"
  description   = "HTTP API Gateway Template"
  protocol_type = "HTTP"

  cors_configuration = {
    allow_headers = ["content-type", "x-amz-date", "authorization", "x-api-key", "x-amz-security-token", "x-amz-user-agent"]
    allow_methods = ["*"]
    allow_origins = ["*"]
  }

  domain_name                 = "${local.subdomain}.${var.hosted_zone}"
  domain_name_certificate_arn = data.aws_acm_certificate.hosted_zone_certificate.arn

  default_stage_access_log_destination_arn = aws_cloudwatch_log_group.logs.arn
  default_stage_access_log_format          = "$context.identity.sourceIp - - [$context.requestTime] \"$context.httpMethod $context.routeKey $context.protocol\" $context.status $context.responseLength $context.requestId $context.integrationErrorMessage"

  integrations = {
    "ANY /" = {
      lambda_arn             = module.lambda_function.this_lambda_function_arn
      payload_format_version = "2.0"
      timeout_milliseconds   = 12000
    }

    "$default" = {
      lambda_arn = module.lambda_function.this_lambda_function_arn
    }
  }

}

```

**Comment by DKurilo** [comment]
I see error:
```
The latest automatic deployment attempt failed.
Deployment attempt failed: Unable to deploy API because no routes exist in this API
Your stage has no deployment attached. Update your API to trigger a new deployment.
```
after such deploy.
I believe the problem is not because `throttling_burst_limit` but because `default_route_settings` and `route_settings` are commented out.
Did someone has the same issue?
Or maybe someone knows workaround?

**Comment by davidkarlsen** [comment]
Fixed in #30

**Comment by davidkarlsen** [comment]
@antonbabenko I think this can be closed with the latest fixes now added.
